whelktool/scripts/cleanups/2020/11/lxl-3413-fix-incorrect-descriptionLanguage/lxl-3413-fix-incorrect-descriptionLanguage.groovy

```diff
@@ -0,0 +1,18 @@
+PrintWriter scheduledForUpdating = getReportWriter("scheduled-updates")
+PrintWriter failedUpdating = getReportWriter("failed-updates")
+
+List ids = new File(scriptDir, 'data-1601623118108.txt').readLines().drop(1).collect{line -> line.split('\t')[1]}
+
+selectByIds(ids) { auth ->
+
+    def descriptionLang = auth.graph[0].descriptionLanguage
+
+    if (!(descriptionLang['@id'] == 'https://id.kb.se/language/swe')) {
+        descriptionLang = ['@id':'https://id.kb.se/language/swe']
```

Comment:
The record is not actually changed here - change `descriptionLang` to `auth.graph[0].descriptionLanguage`

Reply:
Fixed in 7d415dd!